fastchat/conversation.py

```diff
@@ -140,6 +140,17 @@ def get_prompt(self) -> str:
         elif self.sep_style == SeparatorStyle.CHATGLM:
             # source: https://huggingface.co/THUDM/chatglm-6b/blob/1d240ba371910e9282298d4592532d7f0f3e9f3e/modeling_chatglm.py#L1302-L1308
             # source2: https://huggingface.co/THUDM/chatglm2-6b/blob/e186c891cf64310ac66ef10a87e6635fa6c2a579/modeling_chatglm.py#L926
+            if self.name == "chatglm3":
```

Comment:
Could you put a link to the source here?

Reply:
Hi there, 

since chatglm3 uses differenet chat format, I do not think we still need to use the 'CHATGLM' separator style and put a if condition to choice the chat format. But putting the chatglm3 chat format into 'CHATGLM' separator style make sense. I suggest an main contributer to dicide do we have to make an new separator style for it

Reply:
@silk55 could you add a source?